doc/dev/error-message.md

```diff
@@ -0,0 +1,20 @@
+# Error Message
+
+Elektra has a lot of developers and hence the error messages vary in how they are written.
+This guideline should unify the message format so that users will get consistent error messages.
+
+- All error messages start with a capital letter. The only exception are quoted messages at the start such as "'xxxx-xxxx' is syntactically incorrect".
+  If you reference a method name such as `kdbSet()` in the first word, surround it with single quotes.
```

Comment:
For consistencies sake we should either always or never surround method names with quotes. I also can't think of a sentence that cannot be reformulated such that it doesn't start with a method name.